chart/templates/preflight.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    troubleshoot.sh/kind: preflight
+  name: {{ .Release.Name }}-preflight-config
+  namespace: {{ .Release.Namespace | quote }}
+stringData:
+  preflight.yaml: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: Preflight
+    metadata:
+      name: preflight
```

Comment:
I think that here we should also add the namespace or could I be wrong? :thinking: